source/index.html.md

```diff
@@ -333,64 +333,69 @@ Depending on card scheme and merchant category, the name on the card might be
 necessary for approval of credits. It may be provided through the optional
 parameter `card[name]`.
 
-## Recurring payments
+## Series of transactions
 
-Recurring payments enable you to repeatedly charge cardholders without providing
-CSC or other means of cardholder authentication for subsequent payments.
+Clearhaus currently supports the recurring type of subscription billing.
```

Comment:
```suggestion
Clearhaus supports the recurring type of subscription billing.
```
After restructuring away from the enumeration list, _only_ has become an unnecessary filler